docs/IntegrationWithExistingApps.md

```diff
@@ -720,6 +720,8 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
 
 That's it, your activity is ready to run some JavaScript code.
 
+> If your app is targeting the Android `api level 23` or greater, make sure you have, for the development build, the `overlay permission` enabled. You can check it with `Settings.canDrawOverlays(this);`. This is required because, if your app produces an error in the react native component, the error view is displayed above all the other windows. Due to the new permissions system, introduced in the api level 23, the user needs to approve it.
```

Comment:
@SandroMachado I understand that react-native supports Android version 16 and above and I was able to test it using Andriod version 25. I guess the main difference is the whole permissions change and not just the `Draw over other apps` change but for other things as well like accessing contacts or photos etc. So, if a user updates the `targetSdkVersion` to greater than `22`, then he will need more than just instructing the user to allow the `Draw over other apps` permissions. Just once clarification, you said that the native has the same behavior, have you come across any apps where when you first open the app, the `Draw over other apps` screen is shown. I guess this permission is not required for most of the apps and hence I haven't come across it in any android app.